users/models.py

```diff
@@ -131,3 +131,26 @@ def save(self, **kwargs):
             self.last_vote_affirmation = timezone.now()
 
         return super().save(**kwargs)
+
+
+class UserGroup(models.Model):
+    name = models.CharField(max_length=255)
+    location = models.CharField(max_length=255)
+    url = models.URLField('URL')
+
+    TYPE_MEETUP = 'meetup'
+    TYPE_DISTRIBUTION_LIST = 'distribution list'
+    TYPE_OTHER = 'other'
+
+    TYPE_CHOICES = (
+        (TYPE_MEETUP, 'meetup'),
```

Comment:
Perhaps we can capitalize the second element of the tuple?

```py
(TYPE_MEETUP, 'Meetup')
```

I think this may look better on the form.